src/styles/CreateBook.module.css

```diff
@@ -0,0 +1,57 @@
+.createContainer {
+  margin-left: 10%;
+  height: 170px;
+}
+
+.createBookForm {
+  display: flex;
+  flex-direction: row;
+  gap: 50px;
+  width: 100%;
+  padding: 20px;
+  background-color: #fafafa;
+}
+
+.createBookForm label {
+  font-size: 1.2em;
+  background-color: #fafafa;
+}
+
+.createBookForm input,
+.createBookForm select {
+  width: 100%;
+  padding: 10px;
+  font-size: 1em;
+
+  /* border: 1px solid #ddd; */
```

Comment:
## [OPTIONAL]

- [ ] Kindly remove commented code as they will make your code clean and clear